### ISSUE TITLE
easymesh

### ISSUE THREAD

**Issue by legend069** [seud0nym/tch-gui-unhide#130]
hello again :)

i've gotten around to getting another DJA2031 (have 3 now  )
i'm trying to setup this "wifi mesh" thing

flashed all of them to the same firmware
followed - [this](https://github.com/seud0nym/tch-gui-unhide/blob/master/wifi-booster/README.md)

on the two devices that i'm using as extenders they say "1 of 3 SSIDs synced from controller"
the controller(primary gateway) still reports "0 boosters found"
the controller(primary gateway) has the "agent and controller enabled" is there anything else that i'm missing?

there is a log report of "daemon.debug | Multiap_agent | [PLATFORM]   uuid                        : _�E2fOR�Fh&:c* " i think this is for ipv6?
also is there a way to change the logging level without changing everything with the "./bridged-booster" command?

thank you :)

**Comment by seud0nym** [comment]
> on the two devices that i'm using as extenders they say "1 of 3 SSIDs synced from controller"

Try disabling the Agent on the booster devices, save, wait a minute or two, then re-enable. I don't know why it happens, but that seems to resolve it.

> also is there a way to change the logging level without changing everything with the "./bridged-booster" command?

You would have to edit `/etc/config/multiap` and change the logging settings in there. Best to leave logging at the default level 2 - I only used that initially when I was trying to work out why something didn't work.

**Comment by legend069** [comment]
yeah i've tried doing that even restarting it in between agent- enable/disable
is there a process that should be running that i can restart?

**Comment by seud0nym** [comment]
Sorry to ask a basic question, but you do have them connected via ethernet cable, right?

To restart from the command line, it is:
```
/etc/init.d/multiap_controller restart
/etc/init.d/multiap_agent restart
```
(in that order)

**Comment by legend069** [comment]
yes they're all connected by cable. i think i've mixed the cabling method in the house too, (1 AP as typeB another has typeA. but i doubt that would make any difference.)
I've also tried throwing in a crossover cable but that hasn't changed anything.

looking in the "init_management" both "multiap_agent,multiap_agent" have the green light does that mean their process is running?

in order for "wifi mesh" to work do i need to enable "Band Steering" ?
i want to have the 2.4ghz and 5.0ghz on separate SSID's because of "stupid" smart devices

**Comment by seud0nym** [comment]
No, you don't need band steering on. I have it off as well.

Yes, sounds like the processes are running correctly.

Not sure what you mean by type A and type B cables? Are you trying with the devices next to each other, connected by a known working ethernet cable?

The only other thing that I can think of is that some people reported issues if they had a network switch between the two devices. That apparently doesn't work.

**Comment by legend069** [comment]
thanks for the support :)

type a/b cabling methods.
the cable runs between garage/living room/upstairs ect all light up with the right pattern.
the run going from living room to upstairs i think is TypeB and the run going to garage is TypeA. can't remember
the technicolor devies are connected at each end respectively 

i'm going to check it all this weekend and make it all 1 standard. and test it with iperf3
that might be the reason i'm only getting ~100mpbs LAN speeds
although the openspeedtest can still reach ~800mbps. so that confuses me.

![image](https://user-images.githubusercontent.com/40685552/214025294-5077e41c-63fc-4c6d-a405-87e84734667f.png)

i've now added a switch between 1 of them, the other two are still connected to the 1 switch.
with the other two in bridge mode would i be able to use 1 as the controller and the second as the agent 
or would i need to have 1 of them out of bridge mode?

**Comment by seud0nym** [comment]
You shouldn't be using a crossover cable to connect the devices. Just use a standard ethernet cable.

Some people have reported issues when there is a switch between the controller and agent devices. 

You should have 1 controller and as many agents as you need. The normal configuration is for your main internet connected router to be the controller, and any other routers are configured in bridge mode as agents.

**Comment by legend069** [comment]
it's alright.
i've fixed the cabling now but still not having any luck with it.

I'll give up with the easy mesh thing, all the portable devices swap AP's flawlessly anyhow.
just wanted to see what the fuss is about for a "Mesh"

**Comment by seud0nym** [comment]
Thank you very much for the donation. I appreciate it, but I feel I haven't helped you.

The advantages of the mesh systems are that you can near-instantaneous hand-over, and credentials/SSIDs get propagated automatically from the controller. I had been using two DJA0231's meshed for a couple of years, but I never changed the credentials or SSIDs, so that wasn't any benefit. Hand-over was great, though.

I am now running 2 Gen 3 devices, which can't be meshed in the same way. I have better coverage, even with the seconds one just configured as basically a dumb access point. I now have an app on my phone which facilitates the hand-over, so I am not missing much.

Honestly, if the meshing over Wi-Fi was working, I can see the advantages, but for wired it is marginal.

Sorry I couldn't work out what your issue was.

**Comment by legend069** [comment]
i wouldn't worry about it mate :)

it might be my hardware even though all 3 are DJA0231
and on the same firmware - 20.3.c.0389-MR20-RA
i noticed one of them was a bit heavier pulled it open and found it had bigger heat sinks on one of them.
added some photos if you were curious.

i've got all AP's on the same ssid/channel and it feels pretty quick as it is.
it would be nice being able to see what device is connected to what AP (main reason i wanted the mesh thing to work )

[DJA0231.zip](https://github.com/seud0nym/tch-gui-unhide/files/10498210/DJA0231.zip)
